lib/cssom.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the "flow" directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+declare class CSSStyleSheet {
+  cssRules: CSSRuleList;
+  ownerRule: ?CSSRule;
+  deleteRule(index: number): void;
+  insertRule(rule: string, index: number): void;
+}
+
+declare class CSSRule {
+  cssText: string;
+  parentRule: ?CSSRule;
+  parentStyleSheet: ?CSSStyleSheet;
+  type: number;
+  static STYLE_RULE: number;
+  static MEDIA_RULE: number;
+  static FONT_FACE_RULE: number;
+  static PAGE_RULE: number;
+  static IMPORT_RULE: number;
+  static CHARSET_RULE: number;
+  static UNKNOWN_RULE: number;
```

Comment:
Three potential sources of truth: the WedIDL on that page, the table of props underneath or the [CSSOM Working Draft](), none of which fully agree. The table seemed broadest -- I thought perhaps it'd be worth capturing potentially deprecated attributes, since they still might be referenced in code targeting specific browsers. Will remove `UNKNOWN_RULE` if you so recommend.